RELEASES

```diff
@@ -1,11 +1,18 @@
 # Releases
 
-A new release happens every 6 weeks:
+Note: All release >=1.0.0 have been bumped down from vX.0.0 to v0.X.0. This was
```

Comment:
just curious: any links to help read more ? on the reasoning for down versioning ?

Reply:
You can read about versioning here: https://go.dev/ref/mod. It gets quite involved, "Many Go projects released versions at v2 or higher without using a major version suffix before migrating to modules (perhaps before modules were even introduced). These versions are annotated with a +incompatible build tag (for example, v2.0.0+incompatible). See Compatibility with non-module repositories for more information."